packages/sdks-tests/src/e2e-tests/duplicate-attributes.spec.ts

```diff
@@ -3,7 +3,7 @@ import { excludeTestFor, test } from '../helpers/index.js';
 
 test.describe('Duplicate Attributes', () => {
   test('wrapped block has no duplicate attributes', async ({ page, packageName, sdk }) => {
-    test.fail(
+    test.expect(
```

Comment:
I don't understand this change. `test.fail(condition)` means that the test is _guaranteed to fail_ when `condition` is `true`, but `test.expect()` is used for making test assertions.

I think in this case, what you want to do is remove this `test.fail()` command altogether. No need for a replacement